lib/time.js

```diff
@@ -306,40 +318,94 @@ function CronTime(luxon) {
 				) {
 					date = date.plus({ days: 1 });
 					date = date.set({ hour: 0, minute: 0, second: 0 });
+					if (this._forwardDSTJump(0, 0, date)) {
+						const [done, newDate] = this._findPreviousDSTJump(date);
+						date = newDate;
+						if (done) break;
+					}
 					continue;
 				}
 
 				if (!(date.hour in this.hour) && Object.keys(this.hour).length !== 24) {
-					date = date.set({
-						hour: date.hour === 23 && diff > 86400000 ? 0 : date.hour + 1
-					});
+					const expectedHour =
+						date.hour === 23 && diff > 86400000 ? 0 : date.hour + 1;
+					const expectedMinute = date.minute; // expect no change.
+
+					date = date.set({ hour: expectedHour });
 					date = date.set({ minute: 0, second: 0 });
+
+					// When this is the case, Asking luxon to go forward by 1 hour actually made us go forward by more hours...
+					// This indicates that somewhere between these two time points, a forward DST adjustment has happened.
+					// When this happens, the job should be scheduled to execute as though the time has come when the jump is made.
+					// Therefore, the job should be scheduled on the first tick after the forward jump.
+					if (this._forwardDSTJump(expectedHour, expectedMinute, date)) {
+						const [done, newDate] = this._findPreviousDSTJump(date);
+						date = newDate;
+						if (done) break;
+					}
+					// backwards jumps do not seem to have any problems (i.e. double activations),
+					// so they need not be handled in a similar way.
+
 					continue;
 				}
 
 				if (
 					!(date.minute in this.minute) &&
 					Object.keys(this.minute).length !== 60
 				) {
-					date = date.set({
-						minute: date.minute === 59 && diff > 3600000 ? 0 : date.minute + 1
-					});
+					const expectedMinute =
+						date.minute === 59 && diff > 3600000 ? 0 : date.minute + 1;
+					const expectedHour = date.hour + (expectedMinute === 60);
```

Comment:
while I know this technically works I would feel better if these additions were made more explicit with `date.hour + (expectedMinute === 60 ? 1 : 0)`

Reply:
Sure thing! I've found 3 of these in the function and modified them all.

If you think there's any more cases of this implicit boolean conversion that I've missed, feel free to point it out.